servers/quarkus-cli/build.gradle.kts

```diff
@@ -92,20 +92,15 @@ tasks.withType<ProcessResources>().configureEach {
 }
 
 // nessie-quarkus-cli module needs to be adopted before we can generate a native runner
-project.extra["quarkus.package.type"] =
-  if (withUberJar() || project.hasProperty("native")) "uber-jar" else "fast-jar"
+val packageType = if (withUberJar() || project.hasProperty("native")) "uber-jar" else "fast-jar"
+val quarkusBuilderImage = libs.versions.quarkusBuilderImage.get()
```

Comment:
How does Quarkus get to know about this setting?

Reply:
line 132

Reply:
I'm lost :exploding_head: line 132 says `xml.required.set(true)` :smile:

Reply:
Line number changed after rebase 😜